easybuild/easyconfigs/t/TensorFlow/TensorFlow-2.7.1-foss-2021b.eb

```diff
@@ -116,11 +116,6 @@ exts_list = [
     ('opt_einsum', '3.3.0', {
         'checksums': ['59f6475f77bbc37dcf7cd748519c0ec60722e91e63ca114e68821c0c54a46549'],
     }),
-    ('libclang', '13.0.0', {
```

Comment:
tensorflow-io-gcs-filesystem is still included in this easyconfig in the deps

Reply:
Good catch, thanks! Fixed and rebased.